SimCalorimetry/CaloSimAlgos/src/CaloHitResponse.cc

```diff
@@ -127,13 +127,13 @@ CaloSamples CaloHitResponse::makeAnalogSignal(const PCaloHit &hit, CLHEP::HepRan
   if (storePrecise) {
     result.resetPrecise();
     int sampleBin(0);
-    // use 1ns binning for precise sample
-    for (int bin = 0; bin < result.size() * BUNCHSPACE; bin++) {
-      sampleBin = bin / BUNCHSPACE;
+    // use 0.5ns binning for precise sample
+    for (int bin = 0; bin < result.size() * BUNCHSPACE * invdt; bin++) {
+      sampleBin = bin / (BUNCHSPACE * invdt);
       double pulseBit = (*shape)(binTime)*signal;
```

Comment:
Hi @abdoulline, maybe this line needs a factor `dt`?

Reply:
I recall this change by Long  (in https://github.com/cms-sw/cmssw/pull/29310), which we've been discussing with Kevin back then
https://github.com/cms-sw/cmssw/pull/29310/files#diff-ebeb6f4661dfc9b0c622167d87df27846dc8ef5134548f04e4ed411555ec44dbR257

Reply:
@mseidel42   I think so... Line #133  needs "dt" factor, as the sampling is twice more "often" now, so each bin content should be now twice smaller.  Let me try to check/test it explicitly with CaloScan.

Reply:
Ok, thanks for checking! I am also rerunning my local test at the moment.